fedimint-core/src/encoding/btc.rs

```diff
@@ -295,16 +289,14 @@ mod tests {
             ),
         ];
 
-        for (network, magic_legacy_bytes, magic_sane_bytes) in networks {
+        for (network, magic_legacy_bytes, magic_bytes) in networks {
             let mut network_legacy_encoded = Vec::new();
             NetworkLegacyEncodingWrapper(network)
                 .consensus_encode(&mut network_legacy_encoded)
                 .unwrap();
 
-            let mut network_sane_encoded = Vec::new();
-            NetworkSaneEncodingWrapper(network)
-                .consensus_encode(&mut network_sane_encoded)
-                .unwrap();
+            let mut network_encoded = Vec::new();
+            network.consensus_encode(&mut network_encoded).unwrap();
```

Comment:
You can use consensus_encode_to_vec for this.

Reply:
Looks like there are a bunch of places throughout the codebase where we could use `consensus_encode_to_vec()` instead of `consensus_encode().unwrap()`. Since this PR is already approved and ready to merge, I'll fix this in a follow-up.

Reply:
Done in #6364